src/H5Fint.c

```diff
@@ -3800,6 +3854,14 @@ H5F__start_swmr_write(H5F_t *f)
     if (obj_paths)
         H5MM_xfree(obj_paths);
 
+    /* Free access property lists */
+    if (obj_apl_ids) {
+        for (u = 0; u < grp_dset_count; u++)
+            if (obj_apl_ids[u] != H5P_DEFAULT && obj_apl_ids[u] >= 0 && H5I_dec_ref(obj_apl_ids[u]) < 0)
+                HGOTO_ERROR(H5E_ATOM, H5E_CANTDEC, FAIL, "decrementing property list ID failed")
```

Comment:
Should be a DONE_ERROR instead of a GOTO since this is past the done label

Reply:
This is also a problem in develop and needs to be fixed there, too

Reply:
Done and done